RESTore/Extensions.cs

```diff
@@ -35,29 +35,5 @@ internal static string Quote(this string source)
         {
             return "\"" + source + "\"";
         }
-
```

Comment:
Not a negative comment, but a suggestion for a new ticket for Liberator … should we create a Liberator.Utilities library to contain some of these replaced extension methods?